examples/management/managementgroupsubscriptionassociation.yaml

```
@@ -0,0 +1,43 @@
+apiVersion: management.azure.upbound.io/v1beta1
+kind: ManagementGroupSubscriptionAssociation
+metadata:
+  annotations:
+    meta.upbound.io/example-id: management/v1beta1/managementgroupsubscriptionassociation
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    managementGroupIdSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example_sub
+    subscriptionIdSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example_sub
+---
+apiVersion: management.azure.upbound.io/v1beta1
+kind: ManagementGroup
+metadata:
+  annotations:
+    meta.upbound.io/example-id: management/v1beta1/managementgroup
```

```suggestion
    meta.upbound.io/example-id: management/v1beta1/managementgroupsubscriptionassociation
    upjet.upbound.io/manual-intervention: "This resource is skipping because the parent resource was skipped."
```
Could you please fix the example-id?